generators/beamAndVertexGenerator.cc

```diff
@@ -200,7 +419,7 @@ ostream& beamAndVertexGenerator::print(ostream& out) const
 
 	out << "Beam package information:" << endl;
 	out << "    Using beam file ................ ";
-	if(_simpleSimulation) {
+	if(_simulationMode == simpleSimulation) {
```

Comment:
replace if clause by
out << yesNo(_simulationMode == simpleSimulation)
apply this to other cases as well
in principle, the repeated "out <<" could also be omitted